src/components/shared/SelectRecipients/SelectRecipients.tsx

```diff
@@ -35,76 +35,40 @@ export interface SelectRecipientsUIProps<T extends Group | Profile = Group | Pro
     GroupInChipProps?: Partial<GroupInChipProps>
     PersonOrGroupInListProps?: Partial<PersonOrGroupInListProps>
     SelectRecipientsDialogUIProps?: Partial<SelectRecipientsDialogUIProps>
+    children?: React.ReactNode
 }
 
 export function SelectRecipientsUI<T extends Group | Profile = Group | Profile>(props: SelectRecipientsUIProps) {
     const { t } = useI18N()
     const classes = useStylesExtends(useStyles(), props)
-    const { items, selected, onSetSelected } = props
+    const { items, selected, onSetSelected, children } = props
     const isDebugging = useValueRef(debugModeSetting)
     const currentIdentity = useCurrentIdentity()
-    const currentIdentifier = currentIdentity ? currentIdentity.identifier.toText() : ''
     const groupItems = items.filter(x => isGroup(x)) as Group[]
     const profileItems = items.filter(
         x => isProfile(x) && !x.identifier.equals(currentIdentity?.identifier) && x.linkedPersona?.fingerprint,
     ) as Profile[]
     const [open, setOpen] = useState(false)
-    const selectedAsGroups = selected.filter(x => isGroup(x)) as Group[]
```

Comment:
Removing the auto selecting feature?

Reply:
What feature…?

Reply:
What feature…?

Reply:
Auto select followers or following group while selecting friends group.